src/libexpr/primops.cc

```diff
@@ -253,7 +254,7 @@ static RegisterPrimOp primop_import({
       >
       > Unlike some languages, `import` is a regular function in Nix.
       > Paths using the angle bracket syntax (e.g., `import` *\<foo\>*)
-      > are [normal path values](language-values.md).
+      > are normal path values.
```

Comment:
```suggestion
      > are normal [path values](@docroot@/language/values.md#type-path).
```

Does anything speak against making the link useful instead of removing it?

Reply:
I couldn't find this somehow. Problem between keyboard and chair.